src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type TodoListProps = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<TodoListProps> = ({ todo }) => {
+  return (
+    <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      {todo.user && <UserInfo user={todo.user} key={todo.userId} />}
```

Comment:
You need `key` only for the lists, when you map over something
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```